tests/src/common/test_case/wasm.rs

```diff
@@ -0,0 +1,168 @@
+use std::{cell::RefCell, rc::Rc};
+
+use sdk::{
+    cosmwasm_std::{Addr, Api, Binary, BlockInfo, Empty, Querier, Storage, WasmMsg, WasmQuery},
+    cw_multi_test::{
+        AppResponse, Contract as CwContract, CosmosRouter, Wasm as CwWasm, WasmKeeper,
+    },
+    neutron_sdk::bindings::msg::NeutronMsg as InterChainMsg,
+    testing::CwApp,
+};
+
+pub(crate) trait Wasm: CwWasm<InterChainMsg, Empty> + Sized {
+    type CounterPart;
+
+    fn store_code(
+        app: &mut CwApp<Self>,
+        counter_part: &mut Self::CounterPart,
+        code: Box<dyn CwContract<InterChainMsg, Empty>>,
+    ) -> u64;
+}
```

Comment:
could you elaborate on the need for this trait?

Reply:
I made the trait instead of directly plugging the new Wasm module to ensure compatibility with the original one. That way we won't need to use ours when not needed.